packages/client-utils/test/smart-wallet-with-sequence.test.ts

```diff
@@ -0,0 +1,171 @@
+import test from 'ava';
+import { arrayIsLike } from '@agoric/internal/tools/ava-assertions.js';
+import { makeTxSequencer } from '../src/sequence-manager.js';
+import { makeSequencingSmartWallet } from '../src/smart-wallet-with-sequence.js';
+import type { SigningSmartWalletKit } from '../src/signing-smart-wallet-kit.js';
+import { createMockFetchAccount } from './mocks.js';
+
+type SubmitTxResponse = {
+  code: number;
+  height: number;
+  transactionHash: string;
+  sequence: bigint;
+};
+
+class MockSigningSmartWalletKit {
+  private submittedTransactions: Array<{
+    method: string;
+    sequence: bigint;
+  }> = [];
+
+  private networkSequence: () => bigint;
+
+  private shouldSimulateSequenceConflicts: boolean;
```

Comment:
We avoid TypeScript `private` annotations, even in test code... I think these should be actual private fields.
```suggestion
  #submittedTransactions: Array<{
    method: string;
    sequence: bigint;
  }> = [];

  #networkSequence: () => bigint;

  #shouldSimulateSequenceConflicts: boolean;
```